docs/index.md

```diff
@@ -1509,6 +1510,31 @@ Examples:
 
 **The HTML reporter is not intended for use on the command-line.**
 
+## Node.JS native ESM support
+
+Mocha supports writing your tests as ES modules (without needing to use the `esm` polyfill module), and not just using CommonJS. For example:
+
+```js
+// test.mjs
+import {add} from './add.mjs';
+import assert from 'assert';
+
+it('should add to numbers from an es module', () => {
+  assert.equal(add(3, 5), 8);
+});
+```
+
+To enable this you don't need to do anything special. Write your test file as an ES module. In Node.js
+this means either ending the file with a `.mjs` extension, or, if you want to use the regular `.js` extension, by
+adding `"type": "module"` to your `package.json`.
+More information can be found in the [Node.js documentation](https://nodejs.org/api/esm.html).
+
+> Mocha supports ES modules only from Node.js v12.11.0 and above. Also note that
+> to enable this, you need to add `--experimental-modules` when running
+> Mocha. Last, but not least, the ESM implementation Mocha is not yet
+> stable so given that the specification in Node.js may
+> change, this may necessitate changes in Mocha's integration.
+
```

Comment:
In Node >=v13 does it work without that flag?

Reply:
Node >= v13.2.0. And you're right, I should change this text to reflect the stability.